### ISSUE TITLE
Updated to v1.0.123 from v1.0.122, and since then the push notifications stopped working on Android

### ISSUE THREAD

**Issue by vilmosnagy** [humhub/app#260]
Hi there,

Updated to [v1.0.123](https://github.com/humhub/app/releases/tag/v1.0.123) on Android to resolve #244 , and since then the push notifications stopped working for my users.

Probably related, but since the update the app always starts with this screen, asking for the installation's URL:

![image](https://github.com/user-attachments/assets/e1853096-c120-4cb7-b9d0-eec0cc6dcf2b)

The installation seems to be set correctly (and this config worked with the `v1.0.122` app):

![image](https://github.com/user-attachments/assets/5e27fad0-6305-4a76-9b94-eaa11d2d7052)

Could you please help me triage this issue, whether it is related for the update or not?

Thanks,
Vilmos

ps.: can provide the details (installation url, config values) in private message.

**Comment by marc-farre** [comment]
Can you update the Push Notifications (Firebase) module on your HumHub to the latest 2.1.0 version and configure the new fields in the module configuration?
Follow the steps in https://marketplace.humhub.com/module/fcm-push/installation

**Comment by vilmosnagy** [comment]
@marc-farre it's the 2.1.0 FCM Push Notifications module. But to my understanding, the FCM config would be necessary for the PWA push; the push notifications for the mobile app goes through HumHub's servers. At least this page states that (see the underscores):

![image](https://github.com/user-attachments/assets/9b6d4726-d0cf-4f3f-859f-be667de6b19f)

**Comment by marc-farre** [comment]
@vilmosnagy Yes. Can you check the steps on https://community.humhub.com/s/installation-and-setup/wiki/224/mobile-app-push-notifications ?
And mainly the "5. Prerequisites" and "8. Test the configuration"?

**Comment by vilmosnagy** [comment]
@marc-farre see the screenshots below:

![image](https://github.com/user-attachments/assets/780769a1-a67a-4607-bd27-b0e3374b2a07)

![image](https://github.com/user-attachments/assets/7382e194-b737-434c-866a-c4b8fd1364f2)


But I've downgraded on my phone to the `v1.0.122` version (downloaded the apk from the releases tab), and with the same humhub installation, the push notofications started working again. After reinstalling the current, `v1.0.123` version, the push notifications started working.

Give me a couple of days please to test wether it solved the issue or not.

**Comment by marc-farre** [comment]
Thanks for this feedback.
Try also flushing the cache of the mobile app:
1. Android settings
2. Apps
3. Search for "HumHub"
4. Storage
5. Clear data
